README.md

```diff
@@ -16,7 +16,7 @@
 Vega Strike Space Simulation Engine
 ===================================
 
-Vega Strike is Space Flight Simulator that allows a player to explore, trade, and fight in the vast open space. You start in an old beat up cargo ship, with endless possibilities before you and just enough cash to scrape together a life. Yet danger lurks in the space beyond.
+Vega Strike is a Space Flight Simulator that allows a player to explore, trade, and fight in the vast open space. You start in an old beat up cargo ship, with endless possibilities in front of you and just enough cash to scrape together a life. Yet danger lurks in the space beyond.
```

Comment:
In U.S. English, "before you" and "in front of you" mean the same thing. I think we can accept this change, though, if it makes it more understandable to a broader audience.